validate/cli.go

```diff
@@ -12,6 +12,10 @@ import (
 	"github.com/batchcorp/plumber/util"
 )
 
+var (
+	ErrMissingCLIOptions = errors.New("cli options cannot be nil")
```

Comment:
I'd like to add more of these error messages to clean up the `Expect(err.Error()).To(ContainSubstring(*"))` pattern in the tests, but I'm okay with this right now as this will at least give us some space to refactor later and see the tests break if we change things.